BASE/src/ServerTelemetryChannel/Implementation/TelemetryChannelEventSource.cs

```diff
@@ -581,6 +570,17 @@ public void UnexpectedBreezeResponseErrorIndexWarning(int index, string appDomai
             this.WriteEvent(79, index, this.ApplicationName);
         }
 
+        [Event(80, Message = "TransmissionDataLossError. Telemetry items are being lost here as the response code is not in the subset of retriable codes." +
```

Comment:
Can't we just modify the method definition?

Reply:
Are we allowed to change the method name in an EventSource class?
I'll follow up with you offline

Reply:
we can change EventSource method names. Its not part of public API contract.

Reply:
It's not the Public API but I thought it was part of the EventSource Manifest?

Reply:
This is the only information I've been able to find regarding versioning:

> Best Practices
> 7. Attempt to keep EventSource object back compatible and version them appropriately. The default version for an event is 0. The version can be changed by setting [EventAttribute.Version](https://docs.microsoft.com/en-us/dotnet/api/system.diagnostics.tracing.eventattribute.version). Change the version of an event whenever you change the data that is serialized with it. Always add new serialized data to the end of the event declaration, that is, at the end of the list of method parameters. If this isn't possible, create a new event with a new ID to replace the old one.

https://docs.microsoft.com/en-us/dotnet/core/diagnostics/eventsource-instrumentation#best-practices